src/lib/journey/journey.interfaces.ts

```diff
@@ -35,6 +35,7 @@ export interface CallbackMetadata {
 export interface JourneyStore extends Pick<Writable<JourneyStoreValue>, 'subscribe'> {
   next: (prevStep?: StepTypes, nextOptions?: StepOptions) => void;
   reset: () => void;
+  resume: (url: string, nextOptions?: StepOptions) => void;
```

Comment:
Adding a `resume` function here that just wraps `next`, but allows the passing of the `url` argument.